rtgui/thumbnail.cc

```diff
@@ -217,8 +217,16 @@ rtengine::procparams::ProcParams* Thumbnail::createProcParamsForUpdate(bool retu
 
     const CacheImageData* cfs = getCacheImageData();
     Glib::ustring defaultPparamsPath = options.findProfilePath(defProf);
+    bool create = (!hasProcParams() || forceCPB);
```

Comment:
Will that change later? If not, make it `const`. There are a lot of [philosophical reasons](http://www.phoronix.com/scan.php?page=news_item&px=MTI3NDQ) to that, but recently we also stumbled upon a [real issue](https://github.com/Beep6581/RawTherapee/issues/3543) worth considering `const` in most places. One exception I see are by-value parameters as they change the function signature without doing much of substance.